README.MD

```diff
@@ -0,0 +1,68 @@
+#  Foreman Pulp Plugin
+
+Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management
+While this plugin is part of the Foreman project it can only be used with Katello as Foreman is not content aware without the Katello plugin.
+
+## Getting Started
+
+The Foreman project provides documentation on this specific plugin installation from multiple methods, distribution specific RPM, distribution specific DEB and direct manual build from source. 
+The plugin can also be installed as part of the foreman-installer puppet class with the "--[no-]enable-foreman-proxy-plugin-Pulp" option.
+Documentation can be found here for how to install a stand alone smart proxy https://theForeman.org/manuals/$current_foreman_version/index.html#4.3.1SmartProxyInstallation
```

Comment:
We have a `latest` redirect instead of `$current_foreman_version`.

Reply:
updated and tested URL, resolves correctly, useful redirect.